tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,251 @@
+package release
+
+import (
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var myEnvironment = appstudioApi.EnvironmentConfiguration{
+	Env: []appstudioApi.EnvVarPair{
+		{Name: releaseEnvironment, Value: releaseEnvironment},
+	},
+}
+
+var managednamespaceSecret = []corev1.ObjectReference{
+	{Name: redhatAppstudioUserSecret},
+}
+
+var roleRules = map[string][]string{
+	"apiGroupsList": {""},
+	"roleResources": {"secrets"},
+	"roleVerbs":     {"get", "list", "watch"},
+}
+
+var paramsReleaseStrategyDefault = []appstudiov1alpha1.Params{
+	{Name: "extraConfigGitUrl", Value: "https://github.com/scoheb/strategy-configs.git"},
+	{Name: "extraConfigPath", Value: "m6.yaml"},
+	{Name: "extraConfigRevision", Value: "main"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-service-happy-path-with-deployment", Label("release", "defaultBundleWithDeployment"), func() {
```

Comment:
Can we move away from using "happy path". This suite name isn't really descriptive. Can it just be called" release service with deployment" or "release with default bundle" or include where you are deploying into?

Reply:
agree, 
used the  "release-e2e-test with deployment" for better description of this test